bundles/EcommerceFrameworkBundle/OrderManager/V7/OrderAgent.php

```diff
@@ -259,7 +259,7 @@ public function hasPayment()
      */
     public function getCurrency()
     {
-        return $this->environment->getDefaultCurrency();
+        return $this->getOrder()->getCurrency() !== null ? new Currency($this->order->getCurrency()) : $this->environment->getDefaultCurrency();
```

Comment:
Can we make the intention a bit clearer. Tenary operator seems to crowded for this one.
Does this really change the behaviour in the Back Office? I tried to set my Default Currency to `USD`. All my orders are in `EUR`, but this did not make any difference.  

```suggestion
        if($this->getOrder()->getCurrency() !== null) {
            return new Currency($this->getOrder()->getCurrency());
        } 
        return $this->environment->getDefaultCurrency();
```

Reply:
@rliebi the committed changes do not resolve my second question. This actually does not change the back office.

Reply:
what would you think about this solution?

<img width="1158" alt="image" src="https://user-images.githubusercontent.com/2370097/227380670-91aa81d8-6f03-478d-807e-983dd82747ac.png">

styling could be better, sure, but I think this could work for list view.

Reply:
@rliebi Looks a lot better to be but it feels wrong to publish this in a bugfix version.
Would you please rebase your PR to 10.6

Reply:
> @rliebi Looks a lot better to be but it feels wrong to publish this in a bugfix version.
> Would you please rebase your PR to 10.6

I disagree with this one. Nothing changes if there is only one Currency displayed and it fixes if multiple currencies are displayed. No new feature was introduced but a bug was fixed.

I realigned the Totals to one line to make it look more clear:
I am open for suggestions :)
![image](https://user-images.githubusercontent.com/2370097/227622371-64f0be8b-9ee1-4d2a-9a85-b59532a7dc62.png)